### ISSUE TITLE
Android crasher on 6.5.20

### ISSUE THREAD

**Issue by jarredt** [AppsFlyerSDK/appsflyer-react-native-plugin#368]
# Report

## Plugin Version

6.5.20

## On what Platform are you having the issue?

Android

## What did you do?

Released our app with 6.5.20 installed

## What did you expect to happen?

App does not sporadically crash.

## What happened instead?

In our app release that contained 6.5.20 for the first time, we started seeing a number of crashes with the stacktrace included below (which includes a reference to `com.appsflyer.reactnative.RNAppsFlyerModule.onNewIntent(RNAppsFlyerModule.java:759)`. We did not see an error of this kind in prior app releases where we had 6.4.40 installed.

## Please provide any other relevant information.
Here's the full stack trace:

```
java.lang.RuntimeException: Unable to resume activity {com.remitly.passbook/com.remitly.passbook.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.app.Activity.setIntent(android.content.Intent)' on a null object reference
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3808)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3848)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3057)
        at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4940)
        at android.app.ActivityThread.-wrap18(Unknown)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1707)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6797)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:772)

Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.app.Activity.setIntent(android.content.Intent)' on a null object reference
        at com.appsflyer.reactnative.RNAppsFlyerModule.onNewIntent(RNAppsFlyerModule.java:759)
        at com.facebook.react.bridge.ReactContext.onNewIntent(ReactContext.java:254)
        at com.facebook.react.ReactInstanceManager.onNewIntent(ReactInstanceManager.java:506)
        at com.facebook.react.ReactActivityDelegate.onNewIntent(ReactActivityDelegate.java:146)
        at com.facebook.react.ReactActivity.onNewIntent(ReactActivity.java:101)
        at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1330)
        at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1342)
        at android.app.ActivityThread.deliverNewIntents(ActivityThread.java:3131)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3776)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3848)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3057)
        at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4940)
        at android.app.ActivityThread.-wrap18(Unknown)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1707)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6797)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:772)
```

**Comment by yevhenlv** [comment]
Your new release doesn't work with react native 0.67+

**Comment by amit-kremer93** [comment]
Issue is fixed in 6.5.21
